src/main/kotlin/app/permission/model/dto/CreateSnippetInput.kt

```diff
@@ -0,0 +1,12 @@
+package app.permission.model.dto
+
+import jakarta.validation.constraints.NotBlank
+
+data class CreateSnippetInput(
+    @field:NotBlank(message = "Name must not be blank")
+    val name: String?,
+    @field:NotBlank(message = "Snippet Key must not be blank")
+    val snippetKey: String?,
```

Comment:
What would the snippet key represent ? The id of the snippet in the bucket ?

Reply:
Yes